packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -375,17 +375,31 @@ async function fetchLatestSnapshotCore(
 					case "application/json": {
 						let text: string;
 						[text, receiveContentTime] = await measureP(async () =>
-							odspResponse.content.text().catch((error) =>
-								// Parsing can fail and message could contain full request URI, including
-								// tokens, etc. So do not log error object itself.
-								throwOdspNetworkError(
-									"Error while parsing fetch response",
-									fetchIncorrectResponse,
-									odspResponse.content, // response
-									undefined, // response text
-									propsToLog,
+							odspResponse.content
```

Comment:
I think we should simply delete all json code, and not fix it. There is not much value, it's a dead end

Reply:
Will check and do this in other PR.